sentry_sdk/integrations/opentelemetry/span_processor.py

```diff
@@ -45,6 +47,38 @@ def __new__(cls):
 
         return cls.instance
 
+    def __init__(self):
+        # type: () -> None
+        @add_global_event_processor
+        def link_trace_context_to_error_event(event, hint):
```

Comment:
Should there be a check whether the `instrumenter` is set to `otel`?

Reply:
yes. done.